lib/api_repo.js

```diff
@@ -573,32 +574,44 @@ ApiRepo.prototype._buildProtos =
          * @param {?Error} err Failure if specified.
          * @param {string[]} allProtos The list of all proto files.
          */
-        var makeNodeModule = function makeNodeModule(err, allProtos) {
+        var makeNodeModule = function makeNodeModule(err) {
           if (err !== null) {
             findOutputs(err);
             return;
           }
-          var fullPathProtos = _.map(allProtos, function(x) {
-            return path.join(this.repoDir, x);
-          }.bind(this));
 
-          var includePath = _.union(this.includePath, [this.repoDir]);
-          if (!this.opts.nodejsUsePbjs) {
-            makeProtoBasedNodeModule(fullPathProtos, includePath);
-            return;
+          var that = this;
```

Comment:
I would like to do a code-cleanup PR later that will remove the unused `isGoogleApi` flag. Maybe I can also remove all instances of `that = this` at the same time.